bin/spack-install-helper-10-preprocessing

```diff
@@ -116,11 +160,11 @@ case $TARGET_MACHINE in
         #TODO
         ;;
     *)
-        echo "The first argument must be any of '$POSSIBLE_TARGETS'. The rest of the arguments must be valid spack specs"
+        # TODO --arch flag sets the target now
```

Comment:
the TODO is only half valid. It should probably only output `echo "The target argument must be any of '$POSSIBLE_TARGETS'."`